modules/serverless-config-minimal-workloads-serving.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-config-minimal-workloads-serving_{context}"]
+= Configuring Serving for minimal workloads
+
+.Procedure
+
+* You can configure Knative Serving for minimal workloads using the `KnativeServing` Custom Resource (CR):
++
+.A minimal workload configuration in KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 1 <1>
+  workloads:
+    - name: activator
+      replicas: 2 <2>
+      resources:
+        - container: activator
+          requests:
+            cpu: 250m <3>
+            memory: 60Mi <4>
+          limits:
+            cpu: 1000m
+            memory: 600Mi
+    - name: controller
+      replicas: 1 <6>
+      resources:
+        - container: controller
+          requests:
+            cpu: 10m
+            memory: 100Mi <4>
+          limits: <5>
+            cpu: 200m
+            memory: 300Mi
+    - name: webhook
+      replicas: 2 <6>
+      resources:
+        - container: webhook
+          requests:
+            cpu: 100m <7>
+            memory: 60Mi <4>
+          limits:
+            cpu: 200m
+            memory: 200Mi
+  podDisruptionBudgets: <8>
+    - name: activator-pdb
+      minAvailable: 1
+    - name: webhook-pdb
+      minAvailable: 1
+----
+<1> Setting this to `1` scales all system components to one replica.
+<2> Activator should always be scaled to a minimum of two instances to avoid downtime.
+<3> Activator CPU requests should not be set lower than `250m`, as a `HorizontalPodAutoscaler` will use this as a reference to scale up and down.
+<4> Adjust memory requests to the idle values from the previous table. Also adjust memory limits according to your expected load (this might need custom testing to find the best values).
+<5> These limits are sufficient for a minimal-workload scenario, but they also might need adjustments depending on your concrete workload.
+<6> One webhook and one controller are sufficient for a minimal-workload scenario
+<7> Webhook CPU requests should not be set lower than `100m`, as a `HorizontalPodAutoscaler` will use this as a reference to scale up and down.
+<8> Adjust the `PodDistruptionBudgets` to a value lower than `replicas`, to avoid problems during node maintenance.
```

Comment:
would replace HPA and PDB with plain text names (or if you are referring to their CRs, include "CR" after the name in backticks). See e.g. "HorizontalPodAutoscaler is the configuration for a horizontal pod autoscaler" in the API docs